DEPENDENCIES.md

```diff
@@ -757,8 +757,7 @@ graph LR;
 ## npm dependency hierarchy
 
 These are the groups of dependencies in npm that depend on each other.
-Each group depends on packages lower down the chain, nothing depends on
-packages higher up the chain.
+Each group depends on packages lower down the chain, nothing depends on packages higher up the chain.
```

Comment:
This file probably also needs a comment at the top like the template-oss ones.  It's auto generated by a script: https://github.com/npm/cli/blob/latest/scripts/dependency-graph.js#L137-L139